src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double maxPushingCapacity;
+
     public Bulldozer() {
```

Comment:
Don't remove no-args constructors in `Bulldozer` class. It is present which is good, but make sure it has an access modifier 'public' to be consistent and clear about its visibility.